index.html

```diff
@@ -30,16 +41,16 @@ <h2>The Past</h2>
             <h2>Why Did It Have To Be Boxes...</h2>
             
             <div class="boxes">
-                <div class="box">Box 1</div>
-                <div class="box">Box 2</div>
-                <div class="box">Box 3</div>
-                <div class="box">Box 4</div>
-                <div class="box">Box 5</div>
-                <div class="box">Box 6</div>
-                <div class="box">Box 7</div>
-                <div class="box">Box 8</div>
-                <div class="box">Box 9</div>
-                <div class="box">Box 10</div>
+                <div id="one" class="box">Box 1</div>
```

Comment:
```suggestion
                <div class="box" style="color: red;">Box 1</div>
                <div class="box" style="color: blue;">Box 2</div>
                <div class="box" style="color: green;">Box 3</div>
``` You could do something like this then you wouldnt need extra css code in your stylesheet